scripts/evolving.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+$data = file_get_contents('/var/www/log/time_to_evolve.txt');
+$replace = file_get_contents('/var/www/log/replacement.txt');

```

Comment:
Same here.

Reply:
We shouldn't be running this if the files aren't there. This needs to be created in order to run this script.